resources/monitoring/templates/prometheus/rules-1.14/kube-state-metrics.yaml

```diff
@@ -36,10 +37,14 @@ spec:
       for: 15m
       labels:
         severity: critical
+{{- if .Values.defaultRules.additionalRuleLabels }}
+{{ toYaml .Values.defaultRules.additionalRuleLabels | indent 8 }}
+{{- end }}
     - alert: KubeStateMetricsWatchErrors
       annotations:
-        message: kube-state-metrics is experiencing errors at an elevated rate in watch operations. This is likely causing it to not be able to expose metrics about Kubernetes objects correctly or at all.
+        description: kube-state-metrics is experiencing errors at an elevated rate in watch operations. This is likely causing it to not be able to expose metrics about Kubernetes objects correctly or at all.
```

Comment:
Did you see if we need to change other alerts from message to description ?

Reply:
You are right - the upstream rules are inconsistent. Most of them use description though

Reply:
I replaced all remaining messages with descriptions and updated the alertmanager config